tests/components/slack/test_notify.py

```diff
@@ -0,0 +1,66 @@
+"""Test slack notifications."""
+from unittest.mock import Mock
+
+from homeassistant.components.slack.notify import SlackNotificationService
+
+from tests.async_mock import AsyncMock
+
+
+async def test_message_includes_default_emoji():
+    """Tests that default icon is used when no message icon is given."""
+    mock_client = Mock()
+    mock_client.chat_postMessage = AsyncMock()
+    expected_icon = ":robot_face:"
+    service = SlackNotificationService(None, mock_client, "_", "_", expected_icon)
```

Comment:
We want to set up the integration and not interact directly with integration details in the tests. See:
https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations